compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion.hs

```diff
@@ -352,9 +357,16 @@ convertRationalBigNumeric env num denom =
 
     where
         rational = num % denom
+        invalid = unsupported "Large BigNumeric literals are not currently supported. Please construct the number from smaller literals." ()
+
+        mbScale :: Maybe Integer
```

Comment:
@sofiafaro-da if you have an idea for a more direct way to implement this I’m all for it, my brain doesn’t seem to work this morning.

Reply:
Actually yes! Instead of figuring out the scale here, use `numericFromRational` from `DA.Daml.LF.Ast.Numeric`.

Reply:
That's what I should have done the first time. 🙃

Reply:
e526526a6bf59f61e3d0952c9adc396509653f77 which also discovered a bug in `numericE`